### DISCUSSION TITLE
Get a Payout

### DISCUSSION THREAD

**Discussion by hofq** [RainbowMiner/RainbowMiner#1915] [Q&A] [answered]
Hey There,

i am new to mining in general.

How can i get a payout from Mining Pools?

**Comment by RainbowMiner** [comment, marked as answer]
First of all - you get paid automatically, as soon as your mining reward reaches the pool's minimum payout threshold. Each pool has it's own minimum payout threshold, mostly per currency, some pools allow to change that level. So it makes totally sense to have a look onto the homepages of your selected pools, before starting to mine on them. 
For example:
- ZergPool has it's own threshold page: https://zergpool.com/site/payouts
- Zpool says on it's homepage:
  - BTC payouts are processed once a day, in the evening, for balances above 0.0015
  - Payouts for all other currencies are made automatically every 4 hours for balances above 0.05 and balances more than 0.0125 are included in one of the payouts each day.
- Ethermine pays
  - if you mine on the ETH mainnet, if your balance is above the threshold of 0.1 ETH and the network base fee (104 Gwei) is below your Gas Price Limit setting of 40 Gwei. Network tx fees are paid by the miner.
  - if you mine on the POLY network, if your balance is above 0.005 ETH. Network tx fees are paid by the pool
